ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/l2-protocol-forwarding.j2

```diff
@@ -0,0 +1,41 @@
+{# doc - l2 protocol forwarding #}
+{% if l2_protocol.forwarding_profiles | arista.avd.default([]) | length > 0 %}
+
+## L2 Protocol Forwarding
+
+### Forwarding Profiles Summary
+
+| Profile | BFD Per-link RFC-7130 | E-LMI | ISIS | LACP | LLDP | MACSEC | Pause | STP |
+| ------- | --------------------- | ----- | ---- | ---- | ---- | ------ | ----- | --- |
+{%     for profile in l2_protocol.forwarding_profiles | arista.avd.natural_sort('name') %}
+{%         set rows = {} %}
+{%         for protocol in profile.protocols %}
+{%             set proto_name = protocol.name | replace(" ", "_") | replace("-", "_") %}
+{%             do rows.update({proto_name: []}) %}
+{%             if protocol.forward is arista.avd.defined(true) %}
+{%                 do rows[proto_name].append('forward') %}
+{%             endif %}
+{%             if protocol.tagged_forward is arista.avd.defined(true) %}
+{%                 do rows[proto_name].append('tagged forward') %}
+{%             endif %}
+{%             if protocol.untagged_forward is arista.avd.defined(true) %}
+{%                 do rows[proto_name].append('untagged forward') %}
+{%             endif %}
+{%         endfor %}
+{%         set row_bfd = rows["bfd_per_link_rfc_7130"] | arista.avd.default(["-"]) | join(",") %}
+{%         set row_e_lmi = rows["e_lmi"] | arista.avd.default(["-"]) | join(",") %}
+{%         set row_isis = rows["isis"] | arista.avd.default(["-"]) | join(",") %}
+{%         set row_lacp = rows["lacp"] | arista.avd.default(["-"]) | join(",") %}
+{%         set row_lldp = rows["lldp"] | arista.avd.default(["-"]) | join(",") %}
+{%         set row_macsec = rows["macsec"] | arista.avd.default(["-"]) | join(",") %}
+{%         set row_pause = rows["pause"] | arista.avd.default(["-"]) | join(",") %}
+{%         set row_stp = rows["stp"] | arista.avd.default(["-"]) | join(",") %}
+| {{ profile.name }} | {{ row_bfd }} | {{ row_e_lmi }} | {{ row_isis }} | {{ row_lacp }} | {{ row_lldp }} | {{ row_macsec }} | {{ row_pause }} | {{ row_stp }} |
+{%     endfor %}
```

Comment:
So for this one let's create one separate table per profile

like (you may need to fix my typos :) )

```
{%     for profile in l2_protocol.forwarding_profiles | arista.avd.natural_sort('name') %}

#### {{ profile.name }}

| Protocol |Forward | Tagged Forward | Untagged Forward |
| ------- | --------- | ---------------- | ------------------- |
{%         for protocol in profile.protocols %}
{%             set proto_name = protocol.name | replace(" ", "_") | replace("-", "_") %}
{%             set proto_forward =  protocol.forward  | arista.avd.default("-") %}
{%             set proto_tagged_forward =  protocol.tagged_forward | arista.avd.default("-") %}
{%             set proto_untagged_forward =  protocol.untagged_forward | arista.avd.default("-") %}
| {{ proto_name }} | {{ proto_forward }} | {{ proto_tagged_forward }} | {{ proto_untagged_forward }} |
{%         endfor %}
{% endfor %}
```